### DISCUSSION TITLE
Time of max or min point

### DISCUSSION THREAD

**Discussion by m50434** [grafana/grafana#37428] [Dashboards, Panels & Visualizations]
It would be great if there would be an option to show the time of max/mean/min - values beside the values itself in singlestat (or in the the new stat) and gauge.

This is an example:
![grafik](https://user-images.githubusercontent.com/24473301/127739396-8ff17448-b8f5-4286-9613-3adfec3095c0.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
